staging/src/k8s.io/code-generator/cmd/validation-gen/output_tests/supported_resources/root/doc.go

```diff
@@ -0,0 +1,35 @@
+/*
+Copyright 2024 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+// Note: this selects all types in the package.
+// +k8s:validation-gen=*
+// +k8s:validation-gen-scheme-registry=k8s.io/code-generator/cmd/validation-gen/testscheme.Scheme
+// +k8s:validation-gen-test-fixture=validateFalse
+
+// This is a test package.
+package trivial
+
+import "k8s.io/code-generator/cmd/validation-gen/testscheme"
+
+var localSchemeBuilder = testscheme.New()
+
+type T1 struct{}
```

Comment:
What is this test proving?

Reply:
NM, was reviewing by commits and forgot to resolve

Reply:
This ensures that with none of the new tags, that the root resource validates.  All existing tests implicitly cover this.. so I could drop it.  I'll document it better for now and if others feels unnecessary, I'll drop it.